### DISCUSSION TITLE
Do responsible-party and responsible-role serve two different purposes?

### DISCUSSION THREAD

**Discussion by telosBA** [usnistgov/OSCAL#1296] [Q&A] [answered]
The schema includes the use of both responsible-party (`<metadata>`) and responsible-role (`<control-implementation>`). Is this serving the same function?  Why was the naming convention changed?  Is there a use case that we are missing?  Should this be changed to responsible-party for continuity?

**Comment by david-waltermire** [comment, marked as answer]
While `responsible-role` and `responsible-party` have similar models, they differ in one significant way.

- `responsible-role` requires that a `role-id` is referenced, but allows any `party-uuid` references to be omitted. This supports use scenarios where a role is known, but an associated party is not known or cannot be specified by the content author in the [OSCAL use contexts](https://pages.nist.gov/OSCAL/reference/latest/complete/xml-index/#/responsible-role).
- `responsible-party` is similar, but requires that one or more parties are provided using `party-uuid`. This is used in [OSCAL contexts](https://pages.nist.gov/OSCAL/reference/latest/complete/xml-index/#/responsible-party) where the parties must be known and provided.
